### ISSUE TITLE
Issue with automatic backup and restores

### ISSUE THREAD

**Issue by RalfKornmannEnvision** [Azure/service-fabric-issues#1638]
We liked to use the integrated Backup and Restore feature instead of running our own. But something seems not to work as expected.

## Expected Behavior
Show a backup history like in the tutorial: https://docs.microsoft.com/de-de/azure/service-fabric/service-fabric-backuprestoreservice-quickstart-azurecluster ; https://docs.microsoft.com/de-de/azure/service-fabric/media/service-fabric-backuprestoreservice/backup-enumeration.png

## Current Behavior
Backups are created and stored but all of the have Data Loss Version, Configuration Version and Lsn of LastBackup Record of -1. Clicking on the BackupId shows a JSON with {"Error":{"Code":"NotFound", "Message":"Null"}}. Trying to restore one of this backups shows an error message that the BaxckupId is missing even if it has been entered.

## Steps to Reproduce
1. Create an storage account with default settings
2. follow https://docs.microsoft.com/de-de/azure/service-fabric/service-fabric-backuprestoreservice-quickstart-azurecluster using the azure portal and service fabric explorer
3. publish an application with a simple actor service
4. enable backups for this application

**Comment by raunakpandya** [comment]
Please share your cluster resource ID on Azure and the region.

**Comment by RalfKornmannEnvision** [comment]
Thank you for your quick response.

The Cluster Id is d3e4fcfe-6408-4268-a96e-57e839b31619 in westeurope. It's a test cluster that we don't use in production. Therefore I don't care if you break something while checking this issue.

**Comment by raunakpandya** [comment]
Please share me the cluster resource id, which would be in the format subscriptions/<id>/resourcegroups/<rgname>/Microsoft.ServiceFabric/.....

**Comment by RalfKornmannEnvision** [comment]
I am sorry I misunderstood which one you need

/subscriptions/5ad63184-4e72-4cfd-9c24-e9ad708c1dc9/resourcegroups/tsr_backuptest/providers/Microsoft.ServiceFabric/clusters/tsrbackuptestcluster

Is this the correct one?

**Comment by raunakpandya** [comment]
Yes. BTW what do you mean by Clicking on the backupid shows a json with? Where did you find Data Loss Version, Configuration Version and Lsn of LastBackup Record of -1? In the output of backup enumeration? Do you see backups taken properly in the storage account?

**Comment by RalfKornmannEnvision** [comment]
In the service fabric explorer in the backups tap. I have 5 backups for each partition. But each of them has only -1 for the versions and I get the JSON when i click on the backupid of any of them

**Comment by RalfKornmannEnvision** [comment]
Any updates?

**Comment by khandelwalbrijesh** [comment]
@RalfKornmannEnvision have you deleted 'tsrbackuptestcluster'. I could not find it in SFRP explorer of westeurope.

**Comment by RalfKornmannEnvision** [comment]
> @RalfKornmannEnvision have you deleted 'tsrbackuptestcluster'. I could not find it in SFRP explorer of westeurope.

Seems like operations have removed it as it was not used and just costing money. But the issue was not limited to this cluster anyway. If you follow the steps in the first post you should able to reproduce it.

**Comment by khandelwalbrijesh** [comment]
Okay, We will try to reproduce it at our end but since the cluster is deleted therefore we cannot make any promises as to when we will be able to take this up. But we will update you as soon as we get the chance to follow these steps on our side.
Just to confirm more on this, you are only observing this for actor service here. Right?

**Comment by RalfKornmannEnvision** [comment]
> Okay, We will try to reproduce it at our end but since the cluster is deleted therefore we cannot make any promises as to when we will be able to take this up. But we will update you as soon as we get the chance to follow these steps on our side.
> Just to confirm more on this, you are only observing this only for actor service here. Right?

Our application contains only state full actors and stateless services. Therefore I don't know if a state full service would be having this issue, too.
